docs/react-v9/contributing/dev-env.md

```diff
@@ -1,4 +1,100 @@
-// TODO #30642 - A doc explaining how to set up the dev environment.
+This document describes how to set up your development environment and contribute changes to the Fluent UI project. It assumes basic working knowledge with Git and related tools. A typical developer should budget 2 hours from fresh install to their first successful build completion.
 
-Assignee - Mason
-Reviewer - Mak
+- [Basic setup](#basic-setup)
```

Comment:
note: this is created by github automatically, if it's not automated on our part I'd recommend to remove this

Reply:
I'm not following this comment. Any chance it's added to the wrong line?

Reply:
<img width="2574" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/4909716e-429b-4ae7-b54f-ea894470bc92">

Reply:
Oh neat - today I learned!